go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify v1.4.0
-	github.com/tendermint/go-amino v0.15.0
+	github.com/tendermint/go-amino v0.15.1-0.20191003185523-ddfc7fca5e86
```

Comment:
Just pointing out this is the key to accessing the typeInfo's of amino

Reply:
Is this officially released?

Reply:
its just a demo jacked whipped up for use internally so we can use it in building out the typescript libs

Reply:
https://github.com/cosmos/gaia/pull/150/files